akka-docs/src/test/scala/docs/coordination/LeaseDocSpec.scala

```diff
@@ -62,7 +62,7 @@ object LeaseDocSpec {
   }
 }
 
-class LeaseDocSpec extends AkkaSpec(LeaseDocSpec.config) {
+class LeaseDocSpec extends AkkaSpec(LeaseDocSpec.config()) {
```

Comment:
(could also be fixed by removing the `()` on the other side, but no big deal here)